<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QMainWindow" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>OneBook</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAccount_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Create An Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Signin_Customer_Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Sign in as Customer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Signin_Librarian_Button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in as Librarian</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
